src/index.html

```diff
@@ -3,11 +3,169 @@
+      <div class="consultation__info">
+        <h3 class="consultation__title">How long have<br>you do yourself a gift?</h3>
+        <p class="consultation__info">The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+      </div>
+      <form action="#" class="consultation__form" method="GET">
+        <input type="text" name="name" class="consultation__input" placeholder="Name">
+        <input type="text" name="phone" class="consultation__input" placeholder="Phone">
+        <input type="email" name="email" class="consultation__input" placeholder="Email">
+        <button type="button" class="consultation__button">Consultation</button>
+      </form>
+    </section>
+
+    <section class="testimonials">
+      <h2 id="testimonials" class="testimonials__header">Testimonials<span class="crown"></span></h2>
+      <div class="testimonials__cards">
+        <div class="card">
+          <img class="card__img" src="./images/people/1.jpg" alt="photo">
+          <h3 class="card__name">Tim Cook<span class="card__position">CEO</span></h3>
+          <p class="card__descr">Nice work, Certificates National Association of Realtors (USA).</p>
+        </div>
+        <div class="card">
+          <img class="card__img" src="./images/people/2.jpg" alt="photo">
+          <h3 class="card__name">Katherine Adams<span class="card__position">Senior Vice President</span></h3>
+          <p class="card__descr">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.
+          </p>
+        </div>
+        <div class="card">
+          <img class="card__img" src="./images/people/3.jpg" alt="photo">
+          <h3 class="card__name">Jonathan Ive<span class="card__position">Chief Design Officer</span></h3>
+          <p class="card__descr">When my lovely valley of steam rises and half-day sun is above an impermeable.</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="contact">
+      <h2 id="contact" class="contact__header">Contact us<span class="crown"></span></h2>
+      <div class="contact__wrapper">
+        <div class="contact__info">
+          <p class="contact__address">Hello@miami.com<br>+1 234 555-55-55</p>
+          <p class="contact__address">400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401</p>
+        </div>
+        <form action="#" class="contact__form" method="GET">
+          <input class="contact__input" type="text" name="name" placeholder="Name">
+          <input class="contact__input" type="email" name="email" placeholder="Email">
+          <textarea class="contact__message" name="message"  cols="30" rows="10" placeholder="Message"></textarea>
```

Comment:
```suggestion
          <textarea 
            class="contact__message" 
            name="message"  
            cols="30" 
            rows="10" 
            placeholder="Message"
           ></textarea>
```

Reply:
я это исправила, но плейсхолдер перестал работать корректно
нашла это "This one has always been a gotcha for me and many others. In short, the opening and closing tags for the <textarea> element must be on the same line, otherwise a newline character occupies it. The placeholder will therefore not be displayed since the input area contains content (a newline character is, technically, valid content)."
пришлось опять вписать в одну строку